lib/icingadb/redisconnection.cpp

```diff
@@ -226,6 +226,10 @@ void RedisConnection::Connect(asio::yield_context& yc)
 					<< "Trying to connect to Redis server (async) on host '" << m_Host << ":" << m_Port << "'";
 
 				auto conn (Shared<TcpConn>::Make(m_Strand.context()));
+
+				const int optTrue = 1;
+				setsockopt(conn->lowest_layer().native_handle(), IPPROTO_TCP, TCP_NODELAY, reinterpret_cast<const char *>(&optTrue), sizeof(optTrue));
```

Comment:
Why do you explicitly cast there? `setsockopt()` takes a `const void *` so just `&optTrue` should be perfectly fine here.